rpm/del-fixtures-sign.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env sh
+#
+# Delete signatures of all packages recursively in target directory.
+# One RPM utility `rpmsign` must be installed.
+
+set -euo pipefail
+#------------------------------------------------------------------------------
+# Helper Functions
+#------------------------------------------------------------------------------
+
+# Print a message to stdout explaining how to use this script.
+#
+# NOTE: $0 corresponds to the the script's name only in some shells and in some
+# contexts.
+show_help() {
+fmt <<EOF
+usage: del-fixtures-sign.sh <target_dir>
+
+Delete signatures of all packages recursively in <target_dir>
```

Comment:
What are the "all packages" referred to by this sentence? All files ending in `.rpm`, `.drpm` and `.srpm`? Please clarify.